app/views/mailing_list/steps/completed/_first_or_second_year_high_commitment.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>
+  Learn more about <a href="/funding-and-support">funding and the financial support</a> available.
```

Comment:
```suggestion
  Learn more about the <a href="/funding-and-support">funding and financial support</a> available.
```
@philspray I think the link text sounds slightly odd in isolation? (There may be better ways of addressing this than my suggestion)